initfiles/componentfiles/thor/run_thor

```diff
@@ -69,41 +69,65 @@ while [ 1 ]; do
     nohup $instancedir/thormaster_$THORNAME MASTER=$THORMASTER:$THORMASTERPORT 2> /dev/null 1>/dev/null &
 
     thorpid=$!
-    if [ "$thorpid" -ne "0" ]; then 
+    if [[ "$thorpid" -ne "0" ]]; then
         echo thormaster$LCR process started pid = $thorpid
         
         echo $thorpid > $PID_NAME
         echo $thorpid > $PID_NAME_BASE
         wait $thorpid
         errcode=$?
-        if [ "$errcode" -eq "0" ]; then  # allows for CTRL-C or thorstop
-            echo thor exited cleanly - script stopping
-            exit 0
-        fi
-        echo master exited with errorcode = $errcode
-        if [ ! -e $SENTINEL ]; then
-            echo $SENTINEL 'has been removed (1) - script stopping'
+        case $errcode in
+        # TEC_Clean
+        0)  echo "Thormaster ($thorpid) exited cleanly"
             $deploydir/stop_thor $deploydir
```

Comment:
The reason I kept this here was to insure pid file and sentinel file cleanup.  I'll go ahead and remove it and add sentinel file removal into the binary for when we exit with TEC_Clean (in HPCC-13156)